Author: Pino Toscano <pino@debian.org>
Description: Do not fail if cmake does not have policy CMP0071
Forwarded: no
Last-Update: 2018-02-14

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,9 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PAT
 
 cmake_policy(SET CMP0043 NEW)
 cmake_policy(SET CMP0063 NEW)
-cmake_policy(SET CMP0071 NEW)
+if(POLICY CMP0071)
+	cmake_policy(SET CMP0071 NEW)
+endif()
 
 include(KDEInstallDirs)
 include(KDECMakeSettings)
